applications/sofa/gui/qt/viewer/qt/QtViewer.cpp

```diff
@@ -153,6 +154,10 @@ QtViewer::QtViewer(QWidget* parent, const char* name, const unsigned int nbMSAAS
     : QOpenGLWidget(setupGLFormat(nbMSAASamples), parent)
 #endif // defined(QT_VERSION) && QT_VERSION >= 0x050400
 {
+#ifdef __linux__
+    ::setenv("MESA_GL_VERSION_OVERRIDE", "3.0", 1);
+#endif // __linux
+
```

Comment:
Hi and thanks for your comment!
It would indeed be cleaner to check for Intel's hardware before setting this env variable.
The only issue is that I haven't found (nor looked that much yet..) a way of doing so.
Calling glGetString is, to my knowledge, only possible once the OpenGL context is created (?). Thus I would  have to create a dummy gl context, probe the glVendor variable, destroy the dummy context, set the env variable, and then only let Qt create its OpenGL context.
This is quite heavy...
If you guys have an idea how to do this cleanly, I'm all ears :)

Reply:
You are right for the opengl context. 
I don't know how to properly do it. 
Well, it looks overkill :/

Reply:
Ok.. :) Maybe someone else has an idea how to do this in a better way.
I'm setting this PR back to 'to review' until someone finds an acceptable approach to check for the GPU model / vendor